### ISSUE TITLE
front-page.php: Remove static testimonial

### ISSUE THREAD

**Issue by rcass** [redacademy/linc-winter-2018#34]
Since hard coding the front page testimonial is not the best idea, since the client will probably want to change it in the future, we need to make it dynamic.

I would create an additional field that client can check off, and then modify the loop to only display the most recent testimonial set to display on the front page. 
https://github.com/redacademy/linc-winter-2018/blob/ddb93fbd43ab2e2a27faab32d022d171357a0a4f/themes/linc-edge/front-page.php#L102-L106

TL;DR It should not be set to `Chad Myers`. Make it a more dynamic solution

**Comment by calvintyvm** [comment]
I made the testimonial show random on front page.

**Comment by rcass** [comment]
Perfect 👍 That seem like a way better solution that just hard coded to a certain testimonial
